modules/lang/ocaml.el

```diff
@@ -45,12 +52,13 @@
                           (ocp-indent-region (point-min) (point-max)))))
 
 (use-package opam-switch-mode
+  :ensure t
   :after tuareg
   :hook
   (tuareg-mode-hook . opam-switch-mode)
   :bind
   (:map opam-switch-mode-map
-	("C-c C-w" ("set switch" . opam-switch-set-switch))))
+	("C-c C-w" . opam-switch-set-switch)))
```

Comment:
Another broken feature.

Reply:
Huh, very odd. This should add `which-key` hints, but I'm not sure when it broke!

Reply:
It looks like [`:bind` expands to `:init` with `bind-key`s](https://github.com/jwiegley/use-package?tab=readme-ov-file#key-binding), which, as we realized yesterday with `vertico`, does not play well with elpaca.